wrappers/Python/CoolProp/Plots/ConsistencyPlots.py

```
@@ -332,6 +332,34 @@ def consistency_check_singlephase(self):
+        try:
+            from CoolProp.CoolProp import get_fluid_param_string
+            is_pure_fluid = (get_fluid_param_string(self.fluid, "pure") == "true")
+        except Exception:
+            is_pure_fluid = False
+        if self.T_limits_1phase is None and is_pure_fluid and not self.state.has_melting_line():
+            try:
+                state_rhomax = CP.AbstractState(self.backend, self.fluid)
+                state_rhomax.update(CP.QT_INPUTS, 0, self.state.keyed_output(CP.iT_triple))
+                rho_max_1phase = state_rhomax.rhomolar()
+                pc_1phase = self.state.keyed_output(CP.iP_critical)
+            except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)